<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreviewConfigWindow</class>
 <widget class="QWidget" name="PreviewConfigWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Terminal Preview Config</string>
  </property>
  <layout class="QGridLayout" name="main_grid" columnstretch="0,1">
   <item row="4" column="0">
    <widget class="QLabel" name="vertical_margin_label">
     <property name="text">
      <string>Vertical margin:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="word_spacing_label">
     <property name="text">
      <string>Word spacing:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="horizontal_margin_label">
     <property name="text">
      <string>Horizontal margin:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="vertical_margin_spinbox"/>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="line_spacing_spinbox">
     <property name="minimum">
      <double>-99.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="letter_spacing_spinbox">
     <property name="minimum">
      <double>-99.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="horizontal_margin_spinbox"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="line_spacing_label">
     <property name="text">
      <string>Line spacing:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="letter_spacing_label">
     <property name="text">
      <string>Letter spacing:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="word_spacing_spinbox">
     <property name="minimum">
      <double>-99.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="line_numbers_checkbox">
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Show line numbers:</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
